code/modules/power/cell.dm

```diff
@@ -10,6 +10,8 @@
 	throw_speed = 2
 	throw_range = 5
 	w_class = 2
+	var/part_type = "cell" //I blame vehicles
```

Comment:
As far as I can tell, this isn't actually being used. Surely it'd be better to do a type check rather than this?